monitoring/querying-metrics.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: ASSEMBLY
+[id="querying-metrics"]
+= Querying metrics
+include::_attributes/common-attributes.adoc[]
+:context: querying-metrics
+
+toc::[]
+
+You can query metrics to view data about how cluster components and your own workloads are performing.
+
+// Querying metrics
+include::modules/monitoring-about-querying-metrics.adoc[leveloffset=+1]
+//include::modules/monitoring-contents-of-the-metrics-ui.adoc[leveloffset=+2]
+include::modules/monitoring-querying-metrics-for-all-projects-as-an-administrator.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about creating PromQL queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation].
+
+include::modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about creating PromQL queries, see the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus query documentation]. 
+
+include::modules/monitoring-exploring-the-visualized-metrics.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
It looks like this "Additional resources" section is for the preceding module. If so, it's correctly formatted. Otherwise, if it is for the assembly, it should be formatted as described in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies.

Reply:
Thanks. Yes, that "Additional resources" heading is for the preceding module, so no change is necessary